src/lib/isMobilePhone.js

```diff
@@ -70,6 +70,7 @@ const phones = {
   'en-BW': /^(\+?267)?(7[1-8]{1})\d{6}$/,
   'es-AR': /^\+?549(11|[2368]\d)\d{8}$/,
   'es-BO': /^(\+?591)?(6|7)\d{7}$/,
+  'es-BZ': /^(\+501)?[2-8]\d{6}$/,
```

Comment:
```suggestion
  'es-BZ': /^(\+501)?6\d{6}$/,
```
As per the ITU and Belize's public utility commission only numbers starting with 6 are mobile phone numbers. See https://www.itu.int/oth/T0202000016/en